code/modules/discord/discord_helpers.dm

```diff
@@ -0,0 +1,58 @@
+/**
+ * Given a ckey, look up the discord user id attached to the user, if any
+ *
+ * This gets the most recent entry from the discord_link table that is associated with the given ckey
+ *
+ * Arguments:
+ * * lookup_ckey A string representing the ckey to search on
+ */
+/client/proc/discord_lookup_id(lookup_ckey)
+	var/datum/discord_link_record/link = find_discord_link_by_ckey(lookup_ckey)
+	if(link)
+		return link.discord_id
+
+/**
+ * Given a discord id as a string, look up the ckey attached to that account, if any
+ *
+ * This gets the most recent entry from the discord_link table that is associated with this discord id snowflake
+ *
+ * Arguments:
+ * * lookup_id The discord id as a string
+ *
+ * Returns: ckey as string
+ */
+/client/proc/discord_lookup_ckey(lookup_id)
+	var/datum/discord_link_record/link  = find_discord_link_by_discord_id(lookup_id)
+	if(link)
+		return link.ckey
```

Comment:
These two procs don't seem to be used anywhere at the moment. (`discord_lookup_id()` & `discord_lookup_ckey()`)

Reply:
I don't really think that's a problem, I assume there's intended to be a way for this to be called in the future.